test/game_team_test.rb

```diff
@@ -174,23 +174,23 @@ def test_it_can_find_game_ids_by_season
   end
 
   def test_it_find_games_by_season_id
-    assert_equal 12, GameTeam.games_by_season_id(20122013).length
+    assert_equal 12, GameTeam.games_by_season_id("20122013").length
   end
 
   def test_it_can_find_games_by_team_name
-      assert_equal Hash, GameTeam.games_by_team_name(20122013).class
+      assert_equal Hash, GameTeam.games_by_team_name("20122013").class
   end
 
   def test_it_can_find_tackles_by_team
-    result = ({"3"=>77, "6"=>87, "17"=>38, "16"=>72, "9"=>31, "8"=>24, "26"=>46, "28"=>13, "22"=>10})
-    assert_equal result, GameTeam.tackles_by_team("20122013")
+    expected = {"3"=>77, "6"=>87, "17"=>38, "16"=>72, "9"=>31, "8"=>24, "26"=>46, "28"=>13, "22"=>10}
+    assert_equal expected, GameTeam.tackles_by_team("20122013")
   end
 
-  def test_most_tackles
+  def test_team_with_most_tackles
     assert_equal "6", GameTeam.most_tackles("20122013")
   end
 
-  def test_fewest_tackles
+  def test_team_with_fewest_tackles
```

Comment:
this is where the merge error was. I approved since I didn't see any huge changes. Right?